src/logic/game.cc

```diff
@@ -880,6 +880,11 @@ void Game::send_player_set_stock_policy(Building& imm,
 	   get_gametime(), imm.get_owner()->player_number(), imm, ww == wwWORKER, di, sp));
 }
 
+void Game::send_player_toggle_mute(const Building& b, bool all) {
+	send_player_command(
```

Comment:
Do we want this to affect the game state?

Reply:
Message sending (as well as their deletion from the inbox) is done via playercommands, so we need to keep the muting state consistent across clients or there will be desyncs